teensy/main/main.ino

```diff
@@ -81,28 +93,41 @@ int count = 0;
 void print_all_sensors() {
   //prints a single string containing all sensor values
   //to be decoded by python script
-
+  String output = "";
   //expect precision issues to # of digits
   //python script currently expects 2 digits per datum
 
   //GPS
   Serial1.print("^latitude:");
   Serial1.println(myGps->get_latitude());
+  output.concat("^latitude:");
```

Comment:
I added the '^' character for printing to the ESP so that the ESP could find the beginning of the string. With the SD card, I don't think we need to do that, so we can take out these '^'.